src/scales/scale.time.js

```diff
@@ -272,23 +272,51 @@ function determineStepSize(min, max, unit, capacity) {
 	return factor;
 }
 
+function determineMajorUnit(unit) {
```

Comment:
can you leave this method where it was? that will make it easier to compare. I don't think anything has changed besides moving it, but it's a bit hard to tell now. It'll also make it easier to handle conflicts. I'm removing the method in https://github.com/chartjs/Chart.js/pull/6274

Reply:
I moved this because of the eslint error `'determineMajorUnit' was used before it was defined`.

Reply:
Ah, makes sense. Thanks for clarifying